src/scripts/main.js

```diff
@@ -3,92 +3,94 @@
 const Game = require('../modules/Game.class');
 const game = new Game();
 
-const container = document.querySelector('.container');
-const gameField = document.querySelector('.game-field');
-const button = container.querySelector('.button');
-const gameScore = container.querySelector('.game-score');
-
-const fieldRows = [...gameField.querySelectorAll('.field-row')];
-const fieldCells = fieldRows.map((row) => [...row.children]);
-
-const messages = {
-  idle: container.querySelector('.message-start'),
-  lose: container.querySelector('.message-lose'),
-  win: container.querySelector('.message-win'),
+const cells = document.getElementsByClassName('field-cell');
+const score = document.getElementsByClassName('game-score')[0];
+const startAndRestartButton = document.getElementsByClassName('button')[0];
+const messages = document.getElementsByClassName('message');
+
+const bgColorByNumber = {
```

Comment:
you can make it a function that accepts number as an argument and returns `field-cell--${number}`. Also, you can check - `if (!number) { return ''; }`